### PR TITLE
Double-quote all names and user-supplied snippets of text in rdctl output

### PR DESCRIPTION
**Pull request by ericpromislow**
This should hold for the other CLIs as well, but `rdctl` is the chattiest

### CODE REVIEW

File: pkg/rancher-desktop/main/commandServer/settingsValidator.ts [hunk 1]
```diff
@@ -241,7 +241,7 @@ export default class SettingsValidator {
         } else {
           // newSettings[k] should be valid JSON because it came from `JSON.parse(incoming-payload)`.
           // It's an internal error (HTTP Status 500) if it isn't.
-          errors.push(`Setting ${ fqname } should be a simple value, but got <${ JSON.stringify(newSettings[k]) }>.`);
+          errors.push(`Setting "${ fqname }" should be a simple value, but got "${ JSON.stringify(newSettings[k]) }".`);
```
Comment (adamkpickering) [inline pkg/rancher-desktop/main/commandServer/settingsValidator.ts:244]: There are a number of lines in this file (and possibly in other files) that used `<...>` as "quotes" for a variable. Here the style is changed to `"..."` but in most other places it was left as `<...>`.

It doesn't matter to me either way, but we should be consistent with this: all variables should either be switched to the `"..."` quoting style, or left as the `<...>` quoting style.

File: src/go/rdctl/cmd/api.go [hunk 2]
```diff
@@ -77,7 +77,7 @@ func doAPICommand(cmd *cobra.Command, args []string) error {
 		endpoint = fmt.Sprintf("/%s", versionCommand(apiVersion, endpoint))
 	}
 	if apiSettings.InputFile != "" && apiSettings.Body != "" {
-		return fmt.Errorf("api command: --body and --input options cannot both be specified")
+		return fmt.Errorf(`api command: "--body" and "--input" options cannot both be specified`)
```
[the remark this answers is not in the document]
Comment (adamkpickering) [reply src/go/rdctl/cmd/api.go:80]: As a side note, looks like the failure in the E2E tests are at least partially due to this change.

**Review by adamkpickering — APPROVED** [review]
I view messages like this practically: if I (or a user) can read the message and know from it what is going wrong, it has served its purpose. Formatting is mostly irrelevant for this.

Of course, I have my personal preferences, and I wouldn't have made some of the changes here. But I don't think it's worth our time to bikeshed this stuff. I've made a couple of suggestions - it's up to you whether you make related changes.

IMO if we're going to worry about things like this, we should just configure our linters etc to catch/fix this type of issue.

File: src/go/rdctl/cmd/api.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (ericpromislow) [reply src/go/rdctl/cmd/api.go:80]: I'll go with that... I'd rather just quote names of things, meta-things, and user-supplied input.

File: pkg/rancher-desktop/main/commandServer/settingsValidator.ts (hunk 1 above)
Comment (ericpromislow) [reply pkg/rancher-desktop/main/commandServer/settingsValidator.ts:244]: I want to drop `<...>` as a quoting form, and try to use just double-quotes whenever possible.

**Review by adamkpickering — APPROVED** [review]

**Pull request merged**